File: sdkwork-app-sdk-csharp/Models/RecentProjectVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class RecentProjectVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Id { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
        public string? Icon { get; set; }
        public string? Cover { get; set; }
        public string? Type { get; set; }
        public string? Status { get; set; }
        public string? Url { get; set; }
        public string? CreateTime { get; set; }
        public string? UpdateTime { get; set; }
        public int? MemberCount { get; set; }
        public int? FileCount { get; set; }
        public int? Size { get; set; }
        public List<string>? Tags { get; set; }
        public bool? IsPublic { get; set; }
        public bool? IsFavorite { get; set; }
    }
}

File: sdkwork-app-sdk-go/types/recent_project_vo.go
package types

// 最近项目响应
type RecentProjectVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id string `json:"id"`
	Name string `json:"name"`
	Description string `json:"description"`
	Icon string `json:"icon"`
	Cover string `json:"cover"`
	Type string `json:"type"`
	Status string `json:"status"`
	Url string `json:"url"`
	CreateTime string `json:"createTime"`
	UpdateTime string `json:"updateTime"`
	MemberCount int `json:"memberCount"`
	FileCount int `json:"fileCount"`
	Size int `json:"size"`
	Tags []string `json:"tags"`
	IsPublic bool `json:"isPublic"`
	IsFavorite bool `json:"isFavorite"`
}
